.antd-tools.config.js

```diff
@@ -97,8 +97,14 @@ function finalizeDist() {
 function getThemeVariables(options = {}) {
   let themeVar = {
     'hack': \`true;@import "\${require.resolve('antd/lib/style/color/colorPalette.less')}";\`,
-    ...defaultTheme
   };
+
+  if(options.default) {
```

Comment:
Remove `defaultTheme` from up code and config with `options.default` seems a breaking change.

Reply:
Hi! 

Thanks for answering. My idea it's to not make the theme default always on: a solution I needed for a particular project. Since last 5 month then I've working with this and no problems at all. 
Why do you think it would be a breaking change ? 

I would be happy to contribute again with a better solution if you give me a hint.